src/commonMain/kotlin/exchange.dydx.abacus/state/v2/manager/AsyncAbacusStateManagerV2.kt

```diff
@@ -506,50 +508,111 @@ class AsyncAbacusStateManagerV2(
     }
 
     override fun placeOrderPayload(): HumanReadablePlaceOrderPayload? {
-        return adaptor?.placeOrderPayload()
+        try {
+            return adaptor?.placeOrderPayload()
+        } catch (e: Exception) {
+            val error = V4TransactionErrors.error(null, e.toString())
+            trackTransactionError("placeOrderPayload", error)
+            throw e
```

Comment:
All those "..Payload" functions are called by the web FE.. I think web code catches the exception to display the error. 

Here we just catch the exception for logging and re-throw it.